### PR TITLE
fix: update clarinet and tests

### PR DESCRIPTION
**Pull request by nieldeckx**

### CODE REVIEW

**Review by philipdesmedt — CHANGES_REQUESTED** [review]
LGTM! Good work

Any idea why some tests actually changed?

File: clarity/tests/arkadiko-auction-engine-v4-1_test.ts [hunk 1]
```diff
@@ -482,7 +482,7 @@ Clarinet.test({ name: "auction engine: liquidate STX vault without enough USDA t
 
     // Reward data
     call = await liquidationRewards.getRewardData(1);
-    call.result.expectTuple()["share-block"].expectUint(12);
+    call.result.expectTuple()["share-block"].expectUint(15);
```
[the remark this answers is not in the document]
Comment (nieldeckx) [reply clarity/tests/arkadiko-auction-engine-v4-1_test.ts:485]: Had to update a lot of block heights. I guess the deployment of contracts has changed and it now takes more blocks before the tests actually start.

[the remark this answers is not in the document]
Comment (lgalabru) [reply clarity/tests/arkadiko-auction-engine-v4-1_test.ts:485]: So, in order to be a bit more consistent with what would happen with an actual deployment on Devnet/Testnet/Mainnet, contracts are now batched by 25 (tx chaining limit).
you can work around this by generating a deployment plan with
```bash
clarinet deployment generate --simnet
```
and in the `deployments/default.simnet-plan.yaml` file, merge the batches as in turn 
```yaml
- batch
  - transaction 1
  - transaction 2
- batch
  - transaction 3
  - transaction 4
```
into
```yaml
- batch
  - transaction 1
  - transaction 2
  - transaction 3
  - transaction 4
```

clarinet test will use this file by default if it exists

[the remark this answers is not in the document]
Comment (nieldeckx) [reply clarity/tests/arkadiko-auction-engine-v4-1_test.ts:485]: @lgalabru awesome, will give it a go. Thx man! 🙏

**Pull request closed without merging**